### ISSUE TITLE
Version 0.49.2 - No audio in Safari

### ISSUE THREAD

**Issue by bbsixzz** [navidrome/navidrome#2167]
After pulling version v0.49.2 there is no longer any sound.

```
time="2023-02-12T11:16:30Z" level=info msg="Streaming file" artist=Seefeel bitRate=0 cached=false format=raw originalBitRate=245 originalFormat=mp3 re   │
│   questId=8944f75d2267/vJ6mMn6HP2-000009 title=Net transcoding=false user=xxx username=xxx
```
https://github.com/navidrome/navidrome/issues/1932 related?

Safari Version 16.3 (16614.4.6.11.4, 16614)

**Comment by bbsixzz** [comment]
navidrome.toml

```
LogLevel = 'info'
ImageCacheSize = '0'
SessionTimeout = '240h'
UILoginBackgroundUrl = ''
ScanSchedule = '@every 1m'
Scanner.Extractor = 'taglib'

LastFM.Enabled = false
EnableFavourites = true
EnableStarRating = false
EnableLogRedacting = true
EnableMediaFileCoverArt= false
EnableTranscodingConfig = false

ListenBrainz.Enabled = true
ListenBrainz.BaseURL = 'http://10.0.1.2:42010/apis/listenbrainz/1/'
```

I also tried playback on mobilesafari (iPhone 12 mini iOS 16.3), same problem. Although if I put my ear to the speakers I can hear very faint distorted audio playing.

**Comment by deluan** [comment]
Seems that the ReplayGain support is incompatible with Safari, and it was enabled by default in 0.49.2. Try setting `EnableReplayGain = false` in your config, restart the server and reload the UI.

@kgarner7 , this is only reproducible when using the transpiled bundle (`make buildjs`). It does not occur when running `make dev`. Do you have a mac? Can you take a look at this?

**Comment by bbsixzz** [comment]
It works, many thanks for your excellent work.

**Comment by kgarner7** [comment]
Yes, I can confirm that this is a problem with safari. Not sure what it is specifically since I'm not seeing any errors in the console.  Let me check how Safari handles the Web Audio API (in the worst case, it can just be disabled for Safari)...

**Comment by kgarner7** [comment]
So, this problem is due to a weird quirk in how Safari handles Audio Contexts. (See [this SO post](https://stackoverflow.com/questions/57510426/cannot-resume-audiocontext-in-safari) for more details). Unfortunately this will require a bit of kludge to fix

**Comment by deluan** [comment]
But it should work when we have `EnableReplayGain = true` and it is disabled in the preferences, no? Or else I'll have to make it disabled by default in the bug fix release

**Comment by kgarner7** [comment]
It is only disabled when `EnableReplayGain = false`, because otherwise the AudioContext is created. I'll see if I can figure out a proper fix

**Comment by deluan** [comment]
CLosing as the reported problem was fixed in 0.49.3.
